src/burnchains/affirmation.rs

```diff
@@ -0,0 +1,877 @@
+// Copyright (C) 2013-2020 Blockstack PBC, a public benefit corporation
```

Comment:
Can you add a module level rustdoc that's basically just the markdown from the PR description section on affirmation maps? That PR section really helps describe what this module is trying to track and achieve, and without documenting in the code, the PR content will get kind of lost over time.

Reply:
Yup, added